### DISCUSSION TITLE
Download media without reading messages

### DISCUSSION THREAD

**Discussion by LSXAQ** [DamianMorozov/OpenTgResearcher#57] [Ideas]
Hello , 
First I would thank you for this tool specially for someone like me who does not have a great experience in how to run tools and using powershell etc... 
Anyway I Have a suggestion to make this tool better by letting it read and locate media messages and downloading them instantly without reading all messages and defining if a message is empty which takes a lot of time when a group has many messages .
this tool which is now not working had this feature https://github.com/Dineshkarthik/telegram_media_downloader and this tool https://github.com/iyear/tdl which I couldn't figure out how to run it and too complex to me. 

thank you and keep up the good work.

**Comment by DamianMorozov** [comment]
That's a good idea. Uh, thank you. But the thing is, I'm pretty busy with my main job for the rest of the summer, I don't have time for that right now.

**Comment by DamianMorozov** [comment]
I might be able to add this option soon.

**Comment by LSXAQ** [comment]
Well first of all thank u for ur continious support for this tool , as for this feature I wanted to give u some headup related to it , some of older tools had this feature but some people say that it lead them to being banned by telegram.may be it's caused by fast usage of api or something else I don't know really.

**Comment by DamianMorozov** [comment]
Thank you for your positive opinion. I've created an issue #70.

**Comment by DamianMorozov** [comment]
Already implemented.
